apps/etterlatte-hendelser-pdl/src/main/kotlin/hendelserpdl/PersonHendelseFordeler.kt

```diff
@@ -75,41 +75,17 @@ class PersonHendelseFordeler(
 
                 is PdlIdentifikator.Npid -> loggIgnorererNpid(hendelse.hendelseId)
                 is PdlIdentifikator.FolkeregisterIdent -> {
+                    logger.info("Mottok en PDL hendelse (hendelseId=${hendelse.hendelseId})")
```

Comment:
```suggestion
                    logger.info("Mottok en PDL hendelse (hendelseId=${hendelse.hendelseId}, type=${hendelse.opplysningstype})")
```
virker rimelig å legge til typen også i denne loggen

Reply:
![](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExcTB1YXJvYzR1ajUyYzRrMGEwcWsxNjNzYXJzamJ6ZXF0aXo1d3RyciZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/ljaDzjv8cpIpa/giphy.webp)